docs/memfault/Index.rst

```diff
@@ -22,3 +22,4 @@ You can use the integration standalone and without an existing instance of :ref:
    :caption: Subpages:
 
    AWS.rst
+   Azure.rst
```

Comment:
Shouldn't we mention about https://github.com/NordicSemiconductor/asset-tracker-cloud-memfault-azure-js in line 12?